www/src/css/_layout.scss

```diff
@@ -0,0 +1,92 @@
+@import './_var';
+
+.container {
+  margin: auto;
+  max-width: 1400px;
+}
+
+// Main
+// Two sections
+// main.njk
+.main {
+  display: flex;
+  .grid-toc {
+    position: fixed;
+    top: 44px;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    width: 100vw;
+    padding: 1rem;
+    flex-shrink: 0;
+    flex-grow: 0;
+    overflow: auto;
+    z-index: 100;
+
+    @media (min-width: $breakpoint-m) {
+      position: sticky;
+      top: 52px;
+      width: 320px;
+      max-height: calc(100vh - 52px);
+      overflow-y: scroll;
+      -webkit-overflow-scrolling: touch;
+      z-index: 1;
+    }
+  }
+  .grid-body {
+    overflow: hidden;
+    padding-top: 52px;
+    width: 100%;
+
+    @media (min-width: $breakpoint-m) {
+      background: white;
+      flex-grow: 1;
+      min-width: 0;
+      overflow: auto;
+      padding-top: 0;
+    }
+  }
+}
+
+// Content
+// content.njk
+.content {
+  padding: 1rem;
+
+  .content-layout {
+    @media (min-width: $breakpoint-m) {
+      display: flex;
+      flex-direction: row-reverse;
+      margin-bottom: 20vh;
+      margin-left: auto;
+      margin-right: auto;
+      max-width: 1200px;
+      flex-grow: 1;
+    }
+
+    .content-body {
+      background: white;
+      flex-grow: 1;
+      min-width: 0;
+      overflow: auto;
+      padding-top: 0;
+    }
+    .sub-navigation-column {
```

Comment:
minor nit: try and avoid deep nesting CSS if you can help it (keep single-level specificity). This works totally fine for now, but it can come back to be difficult when you try and mix in other styles (but if this was done to override other styles already, then it’s fine—sometimes you gotta do what you gotta do 😛 )

Reply:
What I’ll do is follow a loose [SMACSS](http://smacss.com/book/type-module) style, where rather than nest classes I’ll only extend the classname, like so:

```css
.module {
  /* … */

  &-submodule {
    /* … */
  }
}
```

Compiles to:

```css
.module {}
.module-submodule {}
```

In other words you can use nesting only for organization rather than increasing specificity